src/other/Table.js

```diff
@@ -307,27 +325,40 @@
             .append("td")
         ;
         cells[this.renderHtmlDataCells() ? "html" : "text"](function (d, idx) { 
-            var retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            var retVal;
+            if (!(d instanceof Widget)) {
+                retVal = context.columnPatterns()[idx] ? context.getColumnFormatting(d, idx) : d;
+            }
             return retVal; 
         });
         cells.exit()
             .remove()
-        ;      
+        ;
+
         rows.each(function(tr,trIdx){
-            d3.select(this).selectAll("td").each(function(tdContents,tdIdx){
+            d3.select(this).selectAll("td").each(function(tdContents,tdIdx){    
                 var alignment = context.getColumnAlignment(tdContents);
                 d3.select(this)
-                    .classed("tr-"+trIdx+"-td-"+tdIdx,true)
                     .style("text-align", alignment)
+                    .classed("tr-"+trIdx+"-td-"+tdIdx,true)
                 ;
+                if (tdContents instanceof Widget) {
+                    tdContents.target(this);
+                    tdContents._parentWidget = context;
+                    if (tdContents._class.indexOf("childWidget") < 0) {
+                        tdContents._class += " childWidget";
+                    }
+                    tdContents.render();
```

Comment:
I know the child widgets will render themselves via the widgetArray code, but the problem here is that if they are not rendered now, they will not have a size unless it is explicitly assigned, and the DOM will not know how big to make the table cell containing the widget.  Most charts will need a specific size, but this becomes a problem when you have something like a small nested table, that you want to be displayed in it s full size without scroll bars.  Suggestions?